src/features/AllCitiesList.jsx

```diff
@@ -0,0 +1,58 @@
+import { memo, useState } from 'react';
+import useAllCities from '../hooks/useAllCities';
+import CitiesList from '../components/common/CitiesList/CitiesList';
+import { FETCH_SETTINGS } from '../services/CitiesServise';
+
+const AllCitiesList = ({ initialLimit = 10 }) => {
+  const [limit, setLimit] = useState(initialLimit);
+
+  const increaseLimit = () => {
+    setLimit(prevLimit => {
+      if (prevLimit >= data.length && hasNextPage) {
+        fetchNextPage();
+      }
+      return prevLimit + 10;
+    });
+  };
+
+  const {
+    data,
+    error,
+    isLoading,
+    fetchNextPage,
+    hasNextPage,
+    isFetchingNextPage,
+    total,
+  } = useAllCities({
+    pageSize: 50,
+    sortOrder: FETCH_SETTINGS.SORT.NAME_ASC,
+  });
+
+  return (
+    <div>
+      <CitiesList
```

Comment:
`CitiesList` accepts only following props `{ limit, data: cities = [], error, isLoading }`

Stop passing others not supported props